runtime-native/src/wasm32.rs

```diff
@@ -0,0 +1,48 @@
+use futures::prelude::*;
+use futures::{future::BoxFuture, task::SpawnError};
+// use futures::compat::*;
+
+use std::io;
+use std::net::SocketAddr;
+use std::pin::Pin;
+
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::future_to_promise;
+
+/// The Native runtime.
+#[derive(Debug)]
+pub struct Native;
+
+impl runtime_raw::Runtime for Native {
+    fn spawn_boxed(&self, fut: BoxFuture<'static, ()>) -> Result<(), SpawnError> {
+        use futures01::future::Future;
+        let fut = fut
+            .unit_error()
```

Comment:
Are we sure the `unit_error` call is necessary?

Reply:
Yep, it doesn't compile if we remove it.